templates/zerver/integrations.html

```diff
@@ -1009,6 +1009,43 @@
 
     </div>
 
+    <div id="hellosign" class="integration-instructions">
+
+      <p>
+        Configuring the HelloSign integration is easy, first, create a stream
+        (we recommend naming it <code>hellosign</code>) and add the name of the
+        stream in <code>STREAM_NAME</code>. If no stream is supplied, the default
```

Comment:
>  and add the name of the stream in <code>STREAM_NAME</code>

You should have this later in the doc, just before the URL, maybe?
The user will get confused when reading this.